### DISCUSSION TITLE
Support all & custom time period transactions export in explorer

### DISCUSSION THREAD

**Discussion by MaudSimon** [alephium/alephium-frontend#363] [Feature requests]
Add ability to export all transactions and custom time period.

**Comment by Ryanmr88** [comment]
Thanks Maud! I’ll learn how to write my own feature requests now 😅

**Comment by mvaivre** [comment]
Done!

**Comment by nop33** [comment]
Not sure why this was closed since the explorer is still not supporting the requested features:
- [ ] Support exporting all transactions of an address
- [ ] Support exporting transactions of a user-defined custom period

Opening again in case other users what to upvote it.
